versioned_docs/version-stable/general/faq/is-beaver-builder-a-plugin-or-a-theme.md

```diff
@@ -4,15 +4,14 @@ title: Is Beaver Builder a plugin or a theme?
 sidebar_label: Is Beaver Builder a plugin or a theme?
 ---
 
-Both!
+The Beaver Builder page builder is a plugin, but some of our premium packages include a framework Beaver Builder Theme, as described ehre.
```

Comment:
**Spelling mistake here:**

The Beaver Builder page builder is a plugin, but some of our premium packages include a framework Beaver Builder Theme, as described **ehre**.